contracts/peripheral/YieldDirectorGohm.sol

```diff
@@ -0,0 +1,363 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.10;
+
+import {IERC20} from "../interfaces/IERC20.sol";
+import {IsOHM} from "../interfaces/IsOHM.sol";
+import {IgOHM} from "../interfaces/IgOHM.sol";
+import {IStaking} from "../interfaces/IStaking.sol";
+import {SafeERC20} from "../libraries/SafeERC20.sol";
+import {YieldSplitter} from "../types/YieldSplitter.sol";
+import {OlympusAccessControlled, IOlympusAuthority} from "../types/OlympusAccessControlled.sol";
+
+/**
+    @title YieldDirector (codename Tyche) 
+    @notice This contract allows donors to deposit their sOHM and donate their rebases
+            to any address. Donors will be able to withdraw their principal
+            sOHM at any time. Donation recipients can also redeem accrued rebases at any time.
+ */
+contract YieldDirectorGohm is YieldSplitter, OlympusAccessControlled {
```

Comment:
What do you think of yielddirectorv2 naming scheme instead of this?

Reply:
Like that much better!